Dockerfile

```diff
@@ -0,0 +1,33 @@
+FROM registry.fedoraproject.org/fedora
+
+# For root user
+RUN set -x && \
+  mkdir -p /run/
```

Comment:
Not sure if `/run` is a good directory. It seems to be used already:

```
> podman run --rm -it pvalena/psss-tmt bash
> ls
console     faillock  lock  mdadm   plans    secrets   sudo      user
criu        gluster   log   motd    radvd    sepermit  systemd
cryptsetup  libvirt   lvm   motd.d  rpcbind  setrans   tpm2-tss
```

Also `run` is a bit confusing as the directory is not expected to contain runs (workdir) but plans and tests. Btw, do we need a different directory for regular user versus root? What about just one? Let's say `/home/tmt` or `/home/test` or `/tmt` or `/test`?

Reply:
Right, that's why I did not use it :) before- it works tough ... sure, we can rename it - `/tmt` sounds great!